src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/LogAnalyticTests.ps1

```diff
@@ -0,0 +1,91 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+Test Export Log Analytic Throttled Requests
+#>
+function Test-ExportLogAnalyticThrottledRequests
+{
+    $loc = "West Central US";
```

Comment:
Should use Get-Location everywhere a location is set: https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/Common.ps1#L583

Please ensure you specify all three parameter values.

Reply:
@maddieclayton  this API is only available in this location.  Get-Location gives a different location, so I need to give a specific location for this API.